source/layout.erb

```diff
@@ -31,19 +31,17 @@
       <%= partial "sidebar" %>
 
       <article>
-        <div class="chapter">
```

Comment:
There are some styles that depends on this class. Such as [here](https://github.com/emberjs/guides/blob/redesign/source/stylesheets/base/_lists.scss#L45).

Also, you there are styles that adds the `chapter-mark`, it would be removed and the image as well.